newsfragments/3312.bugfix

```diff
@@ -0,0 +1 @@
+Add your info here
```

Comment:
This needs real text (it becomes user-visible in the changelog of the release)

Reply:
Updated the text in bd20f8d.